resources/puppetlabs/lein-ezbake/template/global/ext/debian/postinst.erb

```diff
@@ -2,6 +2,11 @@
 
 /opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] -%>/scripts/install.sh postinst_deb
 
+# Set some configuration from the project on install
+if [ -z "$2" ]; then
```

Comment:
Where's the value of `$2` coming from?
also, is it meant to be in quotes?

Reply:
`$2` is a variable available in debian packaging, set to the previous version if this is an upgrade, not set if it's a fresh install. Yes, it needs to be in quotes, we're testing to see whether or not this is an empty string